### PR TITLE
Add ambient observed

### PR DESCRIPTION
**Pull request by jmcanterafonseca**

### CODE REVIEW

File: Environment/AirQualityObserved/doc/spec.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
++ `windDirection` : The wind direction expressed in decimal degrees compared to geographic North (measured clockwise), encoded as a Number.
+    + Attribute type: [Number](https://schema.org/(Number)
+    + Default unit: Decimal degrees
+    + Attribute metadata:
+        + `timestamp` : optional timestamp for the observed value. It can be ommitted if the observation time is the same as the one captured
+        by the `dateObserved` attribute at entity level.
+    + Optional 
+
++ `windSpeed` : The observed wind speed in m/s, encoded as a Number
```
Comment (gtorodelvalle) [inline Environment/AirQualityObserved/doc/spec.md]: Missing the dot (`.`) at the end of this line.

File: Environment/AirQualityObserved/doc/spec.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    + Attribute metadata:
+        + `timestamp` : optional timestamp for the observed value. It can be ommitted if the observation time is the same as the one captured
+        by the `dateObserved` attribute at entity level.
+    + Optional 
+
++ `source` : A sequence of characters giving the source of the entity data.
+    + Attribute type: [Text](https://schema.org/Text) or [URL](https://schema.org/URL)
+    + Optional
+
++ `refDevice` : A reference to the device which captured this observation
```
Comment (gtorodelvalle) [inline Environment/AirQualityObserved/doc/spec.md]: Missing the dot (`.`) at the end of this line.

File: Environment/AirQualityObserved/doc/spec.md (hunk 1 above)
Comment (jmcanterafonseca) [inline Environment/AirQualityObserved/doc/spec.md]: fixed https://github.com/Fiware/dataModels/pull/42/commits/d952ce920c84ca19da2af5d0d6ded128ba173abd

File: Environment/AirQualityObserved/doc/spec.md (hunk 2 above)
Comment (jmcanterafonseca) [inline Environment/AirQualityObserved/doc/spec.md]: fixed https://github.com/Fiware/dataModels/pull/42/commits/d952ce920c84ca19da2af5d0d6ded128ba173abd

**Pull request merged**